File: app/src/main/java/org/vaccinationreminder/LoginActivity.java
package org.vaccinationreminder;

import android.annotation.SuppressLint;
import android.os.Bundle;
import android.support.v4.app.FragmentManager;
import android.support.v4.app.FragmentTransaction;
import android.support.v7.app.AppCompatActivity;

public class LoginActivity extends AppCompatActivity {

    @SuppressLint("InflateParams")
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_login);

        LoginActivityLoginFragment loginFragment = new LoginActivityLoginFragment();

        FragmentManager fm = getSupportFragmentManager();
        FragmentTransaction ft = fm.beginTransaction();

        ft.add(R.id.loginActivityFragmentHolder, loginFragment, "LoginActivityLoginFragment");
        ft.commit();

    }
}

File: app/src/main/java/com/adityasubathu/vaccinationreminder/LoginActivity.kt
package com.adityasubathu.vaccinationreminder

import android.annotation.SuppressLint
import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity

class LoginActivity : AppCompatActivity() {
    @SuppressLint("InflateParams")
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_login)
        val loginFragment = LoginActivityLoginFragment()
        val fm = supportFragmentManager
        val ft = fm.beginTransaction()
        ft.add(R.id.loginActivityFragmentHolder, loginFragment, "LoginActivityLoginFragment")
        ft.commit()
    }
}